### PR TITLE
chore:implement api consumption

### PR DESCRIPTION
**Pull request by Riri202**
* Created the CardItem component, imported it into the Card component
* Consumed the pet API in the Card component and passed the returned data as props into imported CardItem component

### CODE REVIEW

File: src/components/Cards.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+function Cards() {
+    const { id } = useParams()
+const [pet, setPet] = useState([]);
+const [loading, setLoading] =useState(false)
+
+const fetchPets = async () => {
+        setLoading(true)
+        try {
+            const response = await fetch(`https://pets-v2.dev-apis.com/pets?id=${id}`);
```
Comment (TheoOkafor) [inline src/components/Cards.js:17]: where's `id` coming from?

File: src/components/Cards.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            const data = response.json();
```
Comment (TheoOkafor) [inline src/components/Cards.js:18]: You need an `await` here

File: src/components/Cards.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            setPet([data.pets]);
+            setLoading(false);
+            
+        }catch (error){
+            alert(error.message);
+        }
+         }
+         useEffect(() => {
+            window.scroll(0, 0)
```
Comment (TheoOkafor) [inline src/components/Cards.js:27]: What does this line do?

**Review by TheoOkafor — COMMENTED** [review]
Check the comments

File: src/components/Cards.js (hunk 2 above)
Comment (Riri202) [reply src/components/Cards.js:18]: Thank you. I missed that.

File: src/components/Cards.js (hunk 3 above)
Comment (Riri202) [reply src/components/Cards.js:27]: To call the fetchPets function as soon as the page loads, before the user scrolls.

File: src/components/Cards.js (hunk 1 above)
Comment (Riri202) [reply src/components/Cards.js:17]: The useParams hook

**Review by TheoOkafor — APPROVED** [review]

**Pull request merged**